src/database/schema/BaseDbSchema.ts

```diff
@@ -2,7 +2,7 @@ import {SQLiteDatabase} from 'react-native-sqlite-storage';
 
 interface BaseDbSchema {
   save(db: SQLiteDatabase): Promise<void>;
-
+  saveIfNotExist(db: SQLiteDatabase): Promise<void>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new method `saveIfNotExist` is added to the `BaseDbSchema` interface. This method should be well-documented, explaining its purpose and how it differs from the existing `save` method. It's also important to ensure that all classes implementing this interface have implemented this new method to maintain consistency.
```diff
+  /**
+   * Save the object into the database if it does not exist already.
+   * @param db - The SQLiteDatabase instance.
+   */
   saveIfNotExist(db: SQLiteDatabase): Promise<void>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->